openeo_udp/eurac_pv_farm_detection/udf_eurac_pvfarm_onnx.py

```diff
@@ -0,0 +1,189 @@
+import functools
+import logging
+import os
+import sys
+import zipfile
+from typing import Dict
+
+import numpy as np
+import requests
+import xarray as xr
+
+
+def _setup_logging():
+    logging.basicConfig(level=logging.INFO)
```

Comment:
I don't think we should promote calling `logging.basicConfig` from UDF code. Backend-side it won't do anything, so this sets wrong expectations